### PR TITLE
clickhouse-cli: init at 0.3.7

### PR DESCRIPTION
**Pull request by bobrik**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 118862` at 071b25a1 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>clickhouse-cli (python38Packages.clickhouse-cli)</li>
    <li>python39Packages.clickhouse-cli</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `stdenv`.
Near pkgs/development/python-modules/clickhouse-cli/default.nix:1:3:
```
  |
1 | { stdenv
  |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/clickhouse-cli/default.nix:30:0:
```
   |
30 |     description = "A third-party client for the Clickhouse DBMS server.";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -1396,6 +1396,8 @@ in {
 
   clickhouse-cityhash = callPackage ../development/python-modules/clickhouse-cityhash {};
 
+  clickhouse-cli = callPackage ../development/python-modules/clickhouse-cli {};
```
Comment (SuperSandro2000) [inline pkgs/top-level/python-packages.nix:1399]:
```suggestion
  clickhouse-cli = callPackage ../development/python-modules/clickhouse-cli { };
```

File: pkgs/development/python-modules/clickhouse-cli/default.nix [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    sha256 = "sha256-fDvUdL6LzgCv6LDmB0R0M7v6BbnbL68p9pHMebP58h8=";
+  };
+
+  propagatedBuildInputs = [
+    click
+    prompt_toolkit
+    pygments
+    requests
+    sqlparse
+  ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/clickhouse-cli/default.nix:27]: checkPhase and pythonImportsCheck missing.

Comment (bobrik) [reply pkgs/development/python-modules/clickhouse-cli/default.nix:26]: There aren't any tests for `checkPhase`.

**Comment by risicle** [comment]
Appears to WFM, macos 10.14

**Review by risicle — APPROVED** [review]

**Pull request merged**